src/ontology/cl-edit.owl

```diff
@@ -32197,12 +32197,14 @@ SubClassOf(obo:CL_4042013 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFr
 
 # Class: obo:CL_4042014 (nasal serous secreting cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37291214") obo:IAO_0000115 obo:CL_4042014 "A serous secreting cell that is part of the nasal cavity.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37291214") obo:IAO_0000115 obo:CL_4042014 "A serous secreting cell that is part of the nasal cavity respiratory epithelium. This cell is part of some submucosal gland.")
```

Comment:
Can you change `some` for `a`? It reads a bit better.

Reply:
of course! on it